filesystem/GD.php

```diff
@@ -140,7 +140,11 @@ public function checkAvailableMemory($filename) {
 		$bytesPerPixel = $bits * $channels;
 
 		// width * height * bytes per pixel
-		$memoryRequired = $imageInfo[0] * $imageInfo[1] * $bytesPerPixel;
```

Comment:
This line was throwing an error. I think the issue is that older version of PHP would silently fail if you tried to access a boolean as an array, while PHP7.4 throws an error.